src/renderer/gl_engine/tvgGlRenderTask.cpp

```diff
@@ -434,3 +434,27 @@ void GlGaussianBlurTask::run()
     }
     GL_CHECK(glEnable(GL_BLEND));
 }
+
+void GlEffectColorTransformTask::run()
+{
+    // const auto vp = getViewport();
+    const auto width = mDstFbo->getWidth();
+    const auto height = mDstFbo->getHeight();
+    // get targets handles and pass to shader
+    GLuint dstCopyTexId = mDstCopyFbo->getColorTexture();
+    GLint srcTextureLoc = getProgram()->getUniformLocation("uSrcTexture");
+    addBindResource({ 0, dstCopyTexId, srcTextureLoc });
+
+    GL_CHECK(glViewport(0, 0, width, height));
+    GL_CHECK(glScissor(0, 0, width, height));
+    // we need to make full copy of dst to intermidiate buffers to be shure, that they are not containe prev data
```

Comment:
typos: 
// we need to make a full copy of dst to intermediate buffers to be sure that they don’t contain prev data.